internal/server/server.go

```diff
@@ -41,3 +55,39 @@ func (s *Server) handleHealthz() http.HandlerFunc {
 		fmt.Fprint(w, string(bytes))
 	}
 }
+
+func logRequestHandler(next http.Handler) http.Handler {
```

Comment:
suggestion: standard logging middleware seems like something we can add to our app template, or perhaps even a separate http middleware library.

Reply:
I plan to have follow up PRs for the app template.